### ISSUE TITLE
[BUG]: Cannot turn on audit log flows after upgrade

### ISSUE THREAD

**Issue by rsch136** [microsoft/powerapps-tools#429]
**Describe the bug**
I recently upgraded the CoE - Audit Logs solution from Version 1.7(?!) to 1.0.0.17, which should be the newest version from July 2020. 

**Component (please tell us which flow or app you are experiencing issues with):**
 - Solution: Audit Logs
 - Version 1.0.0.17
 - Flows: Sync Audit Logs and [Child] Sync Logs
 - using option 2 Azure App registration for custom connector

**To Reproduce**
Upgrade solution. Turn off both flows in order to update start time and end time:
![image](https://user-images.githubusercontent.com/44471683/90614790-6c6fb800-e20b-11ea-86e2-06b18853ba31.png)
Save flows.
Attempt to turn on flows. (Child first, though no order worked for me)

**Expected behavior**
I expect the flows to turn on. Especially if I turn on the child flow first like stated in the documentation: https://docs.microsoft.com/en-us/power-platform/guidance/coe/setup-auditlog#option-2-connect-to-the-audit-log-by-using-an-azure-app-registration

**Screenshots**
![image](https://user-images.githubusercontent.com/44471683/90615024-c96b6e00-e20b-11ea-929e-a01a1378b482.png)
Same error when trying to turn on parent flow.

**Comment by JeneferM-MSFT** [comment]
Can you confirm, did you also complete [these steps from the documentation](https://docs.microsoft.com/power-platform/guidance/coe/setup-auditlog#option-2-connect-to-the-audit-log-by-using-an-azure-app-registration) successfully? 
3. Look at the Initialize tenantID action, and paste in the ID you copied for directory (tenant) ID.
7. Edit the Run only users settings.
8. For both connections (the custom connector and Common Data Service), change the drop-down value to Use this connection (userPrincipalName@company.com). If there is no connection for any of the connectors, go to Data > Connections, and create one for the connector.
9. Select Save, and then close the Flow details tab.

I have seen this error before when people did not complete steps 7-9

**Comment by rsch136** [comment]
Hi Jenefer,
I followed _all_ the steps, including a previous step where I added the custom connector to the dlp (via PowerShell). This added the custom connector to the dlp which caused a slew of other errors in our tenant including an inability to save flows and add sharepoint lists as data sources to powerapps. Only the error message while saving flows gave any usable information specifically that the connector shared_admin-5foffice-20365-20management-20api-xxx was causing an issue. (Do not have any screenshots of this). 
I have removed the connector from the dlp now (also using powershell) and this appears to have solved the problem. The sync audit log flows can be truned on, other flows can be saved, powerapps seems to work fine. I would really appreciate specific documentation for upgrading the CoE managed solutions.
